### PR TITLE
Adding purge schemas plugin

### PR DESCRIPTION
**Pull request by bbejeck**
PR for adding a plugin deleting all schemas

### CODE REVIEW

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -10,7 +10,7 @@
 4. A YAML file named `manifest.yml` that has the following entries. See [cloud-kickstart/manifest.yml](cloud-kickstart/manifest.yml).  The CLI parses the manifest files to generate a list of plugins to install.
     - `description` - A one sentence description of the functionality
     - `dependencies` - What users must have installed to run it.
-5. Add the plugin to the list in the [Available Plugins](README.md#avaiable-plugins) section in the repository README file with a link to its README file.
+5. Add the plugin to the list in the [Available Plugins](README.md#available-plugins) section in the repository README file with a link to its README file.
```
Comment (brianstrauch) [inline CONTRIBUTING.md:13]:
```suggestion
5. Add the plugin to the list in the [Available Plugins](README.md#available-plugins) section in the repository README file with a link to its README file.

```

File: README.md [hunk 2]
```diff
@@ -13,7 +13,9 @@ Here's a list of the current plugins you can install for the confluent CLI:
 
 2. [confluent login headless-sso](confluent-login-headless_sso/README.md)
 
-2. [confluent purge-keys](purge-keys/README.md)
+3. [confluent purge-keys](purge-keys/README.md)
+
+4. [confluent schema-purge](schema-purge/README.md)
```
Comment (brianstrauch) [inline README.md:18]: How do you feel about matching existing command names as best as possible? For instance, `confluent api-key purge` and `confluent schema-registry schema purge`

File: schema-purge/README.md [hunk 3]
```diff
@@ -0,0 +1,24 @@
+### [confluent schema-purge](schema-purge/confluent-schema_purge.py)
+ - Performs a hard delete for all schemas
+   - User prompted to confirm
```
Comment (brianstrauch) [inline schema-purge/README.md:3]:
```suggestion
  - Performs a hard delete for all schemas
  - User prompted to confirm
```

File: schema-purge/README.md [hunk 4]
```diff
@@ -0,0 +1,24 @@
...
+#### Requirements
+  - Python 3 (3.10.9 used for this plugin)  `brew install python3`
+  - [Confluent CLI v3.0.0](https://docs.confluent.io/confluent-cli/current/install.html)
```
Comment (brianstrauch) [inline schema-purge/README.md:6]: Let's suggest that they use the latest minor v3
```suggestion
  - [Confluent CLI v3](https://docs.confluent.io/confluent-cli/current/install.html)
```

**Comment by bbejeck** [comment]
@brianstrauch updated per comments

File: schema-purge/confluent-schema_registry-schema-purge.py [hunk 5]
```diff
@@ -33,10 +33,10 @@ def cli(cmd_args, print_output=False, fmt_json=True):
     return final_result
 
 
-usage_message = 'confluent schema-purge [-h] [--subject-prefix SUBJECT_PREFIX] [--api-key API_KEY] [--api-secret API_SECRET] [--context CONTEXT] [--env ENV] [--secrets-file SECRETS_FILE]'
+usage_message = 'confluent schema-registry schema purge [-h] [--subject-prefix SUBJECT_PREFIX] [--api-key API_KEY] [--api-secret API_SECRET] [--context CONTEXT] [--env ENV] [--secrets-file SECRETS_FILE]'
 
 parser = argparse.ArgumentParser(description='Deletes all schemas permanently'
-                                             'This plugin assumes confluent CLI v3.0.0 or greater',
+                                             ' This plugin assumes confluent CLI v3.0.0 or greater',
```
Comment (brianstrauch) [inline schema-purge/confluent-schema_registry-schema-purge.py:39]: Is this space intentional?

Comment (bbejeck) [reply schema-purge/confluent-schema_registry-schema-purge.py:39]: Yes, it's needed to separate  it from `permanently`

**Comment by bbejeck** [comment]
@brianstrauch reformatted the text in question to one line

**Comment by bbejeck** [comment]
@brianstrauch - rebased this PR and updated for all comments


\cc @davetroiano @sgagniere

File: confluent-schema_registry-schema-purge/confluent-schema_registry-schema-purge.py [hunk 6]
```diff
@@ -0,0 +1,146 @@
+#!/usr/bin/env python3
+
+
+#  Copyright (c) 2023 Confluent
```
[the remark this answers is not in the document]
Comment (bbejeck) [reply confluent-schema_registry-schema-purge/confluent-schema_registry-schema-purge.py:4]: I would say yes - but I think we can make it optional

**Comment by bbejeck** [comment]
@brianstrauch comments addressed

**Review by brianstrauch — APPROVED** [review]

**Pull request merged**